java/ql/src/Security/CWE/CWE-352/SpringCSRFProtection.qhelp

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When you set up a web server to receive a request from a client without any mechanism
+for verifying that it was intentionally sent, then it is vulnerable to attack. An attacker can
+trick a client into making an unintended request to the web server that will be treated as
+an authentic request. This can be done via a URL, image load, XMLHttpRequest, etc. and can 
+result in exposure of data or unintended code execution.</p>
+</overview>
+
+<recommendation>
+<p>Cross-Site Request Forgery (CSRF) protection is enabled by default. Spring's recommendation
```

Comment:
Many apologies for a poor review comment. On re-reading this, I can see that we need to mention Spring in the first sentence (which your original text did).

```suggestion
<p>When you use Spring, Cross-Site Request Forgery (CSRF) protection is enabled by default. Spring's recommendation
```

Reply:
I also missed it. I'll fix it.